File: PRAK205-2510817310018-M. RIFKY ALMADANI.py
tinggi, sisi_miring = map(int, input().split())
alas = ((sisi_miring ** 2) - (tinggi ** 2)) ** 0.5
print("Alas = %d cm" %alas)
print("Tinggi = %d cm" %(tinggi))
print("Keliling = %d cm" %(alas + tinggi + sisi_miring))
print("Luas = %d cm^2" %(0.5 * alas * tinggi))

tinggi = int(input())
sisi_miring = int(input())
alas = ((sisi_miring ** 2) - (tinggi ** 2)) ** 0.5
print("Alas = %d cm" %alas)
print("Tinggi = %d cm" %(tinggi))
print("Keliling = %d cm" %(alas + tinggi + sisi_miring))
print("Luas = %d cm^2" %(0.5 * alas * tinggi))

File: PRAK205-2510817310018-M. RIFKY ALMADANI.c
#include <stdio.h>
#include <math.h>

int main(){
    float tinggi, sisi_miring = 0;
    scanf("%f", &tinggi); 
    scanf("%f", &sisi_miring); 
    float alas = sqrt((sisi_miring * sisi_miring) - (tinggi * tinggi));
    printf("Alas =  %.0f cm \n", alas);
    printf("Tinggi =  %.0f cm \n", tinggi);
    printf("Keliling =  %.0f cm \n", (alas + tinggi + sisi_miring));
    printf("Luas =  %.0f cm^2 \n", (0.5 * alas * tinggi));
    return 0;
}